mne/viz/topomap.py

```diff
@@ -546,18 +559,19 @@ def _get_extra_points(pos, extrapolate, sphere):
         hull_extended = np.unique(hull_extended.reshape((-1, 2)), axis=0)
         new_pos = np.concatenate([hull_extended] + add_points)
     else:
+        assert extrapolate == 'head'
         # return points on the head circle
-        angle = np.arcsin(distance / 2 / head_radius)
+        angle = np.arcsin(distance / 2 / np.mean(radii))
         points_l = np.arange(0, 2 * np.pi, angle)
-        use_radius = head_radius * 1.1
-        points_x = np.cos(points_l) * use_radius + x
-        points_y = np.sin(points_l) * use_radius + y
+        use_radii = radii * 1.1
```

Comment:
Ok, so `radii` are x and y clip radius now, not the head_radius, 👍

Reply:
Yes indeed!